R/LambertWFeaturesWrapper.R

```diff
@@ -0,0 +1,80 @@
+#' @title Gaussianize Numerical Data with a Lambert W Transform
+#'
+#' @description
+#' Takes numeric data and "Gaussianizes" it using an h, hh, or s Lambert W transform.
+#'
+#' @template arg_learner
+#' @param type [\code{character(1)}] \cr
+#' What type of non-normality: symmetric heavy-tails "h" (default),
+#'  skewed heavy-tails "hh", or just skewed "s".
```

Comment:
That is a better name, but in the LambertW package skewed heavy distributions are called 'hh' because it is an hh distribution.